tests/token/test_rest.py

```diff
@@ -3,23 +3,27 @@
 
 import pytest
 
-from spylib import Store
+from spylib import Token
 from spylib.exceptions import ShopifyCallInvalidError
 
-from ..shared import MockHTTPResponse
+from ..token_classes import MockHTTPResponse, app_information, offline_token_data
 
 
 @pytest.mark.asyncio
 async def test_store_rest_happypath(mocker):
-    store = Store(store_id='TEST', name='test-store', access_token='Te5tM3')
+    token = Token(
```

Comment:
I'm confused ... Still not using `.load`   :persevere:

Reply:
Oh because there is no load method on the base Token class. This shouldn't even happen because you aren't supposed to instantiate it as it's abstract. I did this based on a previous comment that we should be testing the functionality of the ABC using the ABC.

Reply:
Ok that's where the misunderstanding is. I don't remember what I said but I didn't mean to test `Token` directly. You should use your `OfflineToken` or `OnlineToken` derived from `OfflineTokenABC and `OnlineTokenABC` respectively and use them as a user would. That will indirectly but in real conditions test `Token`.

Reply:
Ok great, so just nuke this test?